metrics-operator/controllers/common/providers/dynatrace/dynatrace_dql.go

```diff
@@ -137,27 +199,42 @@ func (d *keptnDynatraceDQLProvider) ensureDTClientIsSetUp(ctx context.Context, p
 	return nil
 }
 
-func (d *keptnDynatraceDQLProvider) postDQL(ctx context.Context, query string) (*DynatraceDQLHandler, error) {
+func (d *keptnDynatraceDQLProvider) postDQL(ctx context.Context, metric metricsapi.KeptnMetric) ([]byte, int, error) {
 	d.log.V(10).Info("posting DQL")
 	ctx, cancel := context.WithTimeout(ctx, 30*time.Second)
 	defer cancel()
 
-	values := url.Values{}
-	values.Add("query", query)
+	path := "/platform/storage/query/v0.7/query:execute"
```

Comment:
let's make the base path `/platform/storage/query/v1/query` a constant in this file and use that for the `:execute` and `:poll` requests